### PR TITLE
Add helper to TestCase to get  a fully qualified reference to a method that is defined on the test class

### PR DESCRIPTION
**Pull request by lucasmichot**
- [ ] Added or updated tests
- [ ] Added Docs for all relevant versions
- [ ] Updated the changelog

**Related Issue/Intent**

Factorise tests resolvers

**Changes**

**Breaking changes**

### CODE REVIEW

File: tests/TestCase.php [hunk 1]
```diff
@@ -288,4 +288,15 @@ protected function placeholderQuery(): string
...
+
+    /**
+     * Get the test resolver.
+     *
+     * @param  string  $method
+     * @return string
+     */
+    protected function getResolver(string $method = 'resolve'): string
+    {
+        return addslashes(static::class)."@{$method}";
```
Comment (spawnia) [inline tests/TestCase.php]:
```suggestion
        return addslashes(static::class).$method;
```

File: tests/TestCase.php [hunk 2]
```diff
@@ -288,4 +288,15 @@ protected function placeholderQuery(): string
...
         ';
     }
+
+    /**
+     * Get the test resolver.
+     *
+     * @param  string  $method
+     * @return string
+     */
+    protected function getResolver(string $method = 'resolve'): string
```
Comment (spawnia) [inline tests/TestCase.php]: `get` indicicates this returns a property, so maybe we can rename that, ideas:
- qualifyResolverMethod
- qualifiedTestResolver

Open to suggestions.

File: tests/TestCase.php (hunk 1 above)
Comment (spawnia) [reply tests/TestCase.php]: Oops, let's not do that.

File: tests/TestCase.php (hunk 2 above)
Comment (lucasmichot) [reply tests/TestCase.php]: 👍

**Pull request merged**